.travis.yml

```diff
@@ -16,7 +16,7 @@ _linux: &linux
   os: linux
 _osx: &osx
   os: osx
-  osx_image: xcode10.1
+  osx_image: xcode11.2
```

Comment:
Can we keep the old Xcode?

Reply:
Sure, why not. As right now the issue is gone.